OpenEdXMobile/res/values/strings.xml

```diff
@@ -420,6 +420,10 @@
     <string name="assessment_soon">Soon</string>
     <!-- Label to notify user to rotate device for viewing a video in full screen -->
     <string name="assessment_rotate_for_fullscreen">Rotate your device to view this video in full screen.</string>
+    <!-- Label showing only when youtube in-app player produce error -->
+    <string name="assessment_unable_to_play_video" tools:ignore="MissingTranslation">Player Error</string>
+    <!-- Error message showing only when youtube in-app player produce error -->
+    <string name="assessment_unable_to_play_video_message" tools:ignore="MissingTranslation">Unable to play this video</string>
```

Comment:
We can be more elaborative in the message like
`Unable to play this video at the moment.\nYou can view it on youtube app or watch it later.`
And show 'Watch later' button instead of 'Cancel'
Get it approved from Greg or Khalid.

Reply:
@shamck which error message sounds better to you when the in-app YouTube player is unable to play a video for whatever reason:

Option 1: Unable to play this video
Option 2: Unable to play this video at the moment. You can view it on the YouTube app or watch it later.

> And show 'Watch later' button instead of 'Cancel'

Watch later gives a different context in terms of how YouTube uses it (like queuing a video for the future). I would suggest we go with `OK` as the text for the button to dismiss the error dialog. Thoughts @shamck?

Reply:
With regards to the error message, how about this slight variant of Option 2:
"Unable to play this video. You can view it on the YouTube app or try again later." 

Yes, I agree, let's go with "OK" as the button text

Reply:
Sounds good to me @shamck. Thanks!